Sources/Extensions/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -43,6 +43,43 @@ public extension Array where Element: FloatingPoint {
 	
 }
 
+// MARK: Methods (String)
+public extension Array where Element == String {
+    
+    /// SwifterSwift: Takes an array of strings and generate two lists:
+    ///               One is derived from their first character which will serve as their section label.
+    ///               Second is the array items now grouped based on their corresponding section (first letter).
+    ///
+    ///    ["Jason", "Alice", "Jack"].groupToSections() -> (["A", "J"], [["Alice"], ["Jack", "Jason"]])
+    ///
+    /// - Returns: A tuple consists of the sections list, and its grouped items.
+    public func groupToSections() -> ([String], [[String]]) {
+        // https://stackoverflow.com/questions/42981122/swift-map-array-of-objects-alphabetically-by-namestring-into-separate-letter
+        var items = self
+        
+        items = items.sorted(by: { $0 < $1 })
+        
+        let sections = Array(Set(items.map { String($0.first!) })).sorted(by: { $0 < $1 })
```

Comment:
Will be crash if an item is an empty string "".  Using force unwrapped is bad practice. And the same case in line 68